Cannot find name 'AsyncIterable'

The following error is thrown

```
c:/dev/burkeholland/ignite-app/node_modules/@azure/cosmos/lib/queryIterator.d.ts(68,25): error TS2304: Cannot find name 'AsyncIterable'.
```

This is resolved by adding the following to `compilerOptions` in the tsconfig.json...

```
"lib": [
  "esnext"
]
```

Perhaps we can mention something in the docs about this? Or maybe I'm doing something else wrong?

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -13,7 +13,7 @@
     "target": "es6",
     "sourceMap": true,
     "newLine": "LF",
-    "lib": ["es6", "esnext.asynciterable"]
+    "lib": ["es6", "esnext"]
   },
   "include": ["./src/**/*", "./test/**/*.spec.ts"],
   "exclude": ["node_modules"]

